scripts/process_schemas.py

```diff
@@ -282,6 +282,9 @@ def __init__(self, args):
         string = "schemas-{0}".format(self.version[1:])
         self.schemaDir = os.path.join(self.tmpDir, string)
         self.avroJar = os.path.join(self.schemaDir, "avro-tools.jar")
+        if args.inputSchemasDirectory is not None:
+            self.schemaDir = args.inputSchemasDirectory
```

Comment:
Agree. I saw the problem with my initial approach – lots of generated intermediate files – as soon as I started playing with this code on my local clone of the schema repo.